src/components/loading/_loading_elastic.scss

```
@@ -0,0 +1,79 @@
+.euiLoadingElastic {
+  position: relative;
+  display: inline-block;
+}
+
+.euiLoadingElastic--medium {
+  width: $euiSize;
+}
+
+.euiLoadingElastic--large {
+  width: $euiSizeL;
+}
+
+.euiLoadingElastic--xLarge {
+  width: $euiSizeXL;
+}
+
+
+.euiLoadingElastic path {
+  animation-name: scaleElastic;
+  animation-fill-mode: forwards;
+  animation-direction: alternate;
+  transform-style: preserve-3d;
+  animation-duration: 1s;
+  animation-timing-function: cubic-bezier(0, .63, .49, 1);
+  animation-iteration-count: infinite;
+  transform-origin: 50% 50%;
+}
+
+.euiLoadingElastic path:nth-of-type(1) {
+  animation-delay: 0s;
+}
+
+.euiLoadingElastic path:nth-of-type(2) {
+  animation-delay: .035s;
+}
+
+.euiLoadingElastic path:nth-of-type(3) {
+  animation-delay: .125s;
+}
+
+.euiLoadingElastic path:nth-of-type(4) {
+  animation-delay: .155s;
+}
+
+.euiLoadingElastic path:nth-of-type(5) {
+  animation-delay: .075s;
+}
+
+.euiLoadingElastic path:nth-of-type(6) {
+  animation-delay: .06s;
+};
+
+
+@keyframes scaleElastic {
```

```suggestion
@keyframes euiLoadingElastic {
```

Best to name these the same as your component.